core/build.gradle

```diff
@@ -77,7 +77,7 @@ dependencies {
     runtime project(':cloud-reactor')
     runtime project(':cloud-openstack')
     compile project(':cloud-arm')
-    runtime project(':cloud-gcp')
+    compile project(':cloud-gcp')
```

Comment:
this is incorrect, it must be runtime dependency.  This is valid for all providers (arm, gcp, aws)